scrapy/mail.py

```diff
@@ -129,12 +129,29 @@ def _sendmail(self, to_addrs, msg):
         # Import twisted.mail here because it is not available in python3
         from twisted.internet import reactor
         from twisted.mail.smtp import ESMTPSenderFactory
-        msg = BytesIO(msg)
+        from twisted import version as twisted_version
+        from distutils.version import Version
+
         d = defer.Deferred()
-        factory = ESMTPSenderFactory(
-            self.smtpuser, self.smtppass, self.mailfrom, to_addrs, msg, d,
-            heloFallback=True, requireAuthentication=False, requireTransportSecurity=self.smtptls,
-        )
+
+        factory_config = {
```

Comment:
I would make a dict only for keyword arguments, not for all of them, but I guess that's a style question

Reply:
I agree, it makes it shorter as well. I will adjust it